test/quantization/test_quantized_op.py

```diff
@@ -2532,6 +2532,7 @@ def test_qlstmGRU(self, num_batches, input_size, hidden_size,
                 if torch.backends.quantized.engine == 'qnnpack' and dtype == torch.float16:
                     continue
 
+                print(rnn_type, dtype, num_directions, torch.backends.quantized.engine)
```

Comment:
remove?

Reply:
Yes, currently the test is failing on certain machines, not able to repro locally, will remove once I figure out the cause :-)